contracts/core/Registry.sol

```diff
@@ -112,61 +108,69 @@ contract Registry is IRegistry, Initializable, Native, AccessControlUpgradeable,
     ///               for each profile.
     /// @param _name The name of the profile
     /// @param _metadata The metadata of the profile
-    /// @param _owner The owner of the profile
+    /// @param _owners The owners of the profile
     /// @param _members The members of the profile (can be set only if msg.sender == _owner)
     /// @return The ID for the created profile
     function createProfile(
         uint256 _nonce,
         string memory _name,
         Metadata memory _metadata,
-        address _owner,
+        address[] memory _owners,
         address[] memory _members
     ) external returns (bytes32) {
         // Generate a profile ID using a nonce and the msg.sender
-        bytes32 profileId = _generateProfileId(_nonce, _owner);
+        bytes32 profileId = _generateProfileId(_nonce, msg.sender);
 
         // Make sure the nonce is available
         if (profilesById[profileId].anchor != address(0)) revert NONCE_NOT_AVAILABLE();
 
-        // Make sure the owner is not the zero address
-        if (_owner == address(0)) revert ZERO_ADDRESS();
-
         // Create a new Profile instance, also generates the anchor address
         Profile memory profile = Profile({
             id: profileId,
             nonce: _nonce,
             name: _name,
             metadata: _metadata,
-            owner: _owner,
             anchor: _generateAnchor(profileId, _name)
         });
 
         profilesById[profileId] = profile;
         anchorToProfileId[profile.anchor] = profileId;
 
-        // Assign roles for the profile members
-        uint256 memberLength = _members.length;
+        bytes32 PROFILE_MEMBER_ROLE = profileId;
+        bytes32 PROFILE_OWNER_ROLE = keccak256(abi.encodePacked(profileId, "owner"));
+
+        // Assign roles for the profile owners
+        _grantRole(PROFILE_OWNER_ROLE, msg.sender);
```

Comment:
Why do we give the owner role to the msg.sender?

Reply:
For simplicity of the code, and if you provide an empty array of owners, no error is thrown and you can have only one owner.